Components/Utils/ModelRenderer.cs

```diff
@@ -125,13 +125,27 @@ public HtmlString UnilingualPreFooter()
 
         public HtmlString PreFooter()
         {
+            Feedback feedback = new Feedback()
```

Comment:
As discussed, it would be something like this:
- If Text, Section OR Theme is specified, set the new properties  (text and Href based on current culture)
- Else set legacy url (based on current culture)